### DISCUSSION TITLE
Welcome to OnenoteTaggingKit Discussions!

### DISCUSSION THREAD

**Discussion by WetHat** [WetHat/OnenoteTaggingKit#42] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roy6n23** [comment]
Hello,

May I ask if the tag is compatible with Microsoft 365?

My version is

Microsoft® OneNote® 适用于 Microsoft 365MSO (版本 2110 Build 16.0.14527.20270) 64 位

**Reply by WetHat** [reply]
The TaggingKit is compatible with all OneNote **desktop** versions starting with OneNote 2013.  All desktop versions which are part of Office 365 are supported too. The WIndows 10 App Store version is not supported see issue #6 .

**Reply by roy6n23** [reply]
Thank you so much for your reply.
After trying lots of time, I still get "not loaded. A runtime error occurred during the loading of the COM add-in." message when trying to add the add-in in OneNote (Microsoft 365 Apps for enterprise).

**Comment by WetHat** [comment]
There is a service called 'dllhost' which is needed to load .net add-ins (which the TaggingKit is). Sometimes that service gets stuck. The easiest way to recover  from  that is to **reboot** the computer and then open OneNote again. If the TaggingKit does not show up after that please, try the steps in   [Tagging Kit Not Shown in Ribbon](https://github.com/WetHat/OnenoteTaggingKit/wiki/Tagging-Kit-Not-Shown-in-Ribbon).

Let me know how that goes.

**Reply by roy6n23** [reply]
Hello,

I did reboot my computer after install. But after I add the add-in in settings, and reopen the OneNote, I got "We are sorry. OneNote is cleaning up from the last time it was open. Please wait". No matter how many times I open OneNote, I always got the notice, unless I uninstall the TaggingKit using "choco uninstall onenote-taggingkit-addin.install". Maybe it is not compatible with OneNote 2021?

Thank you for your patience.

**Reply by WetHat** [reply]
The TaggingKit has been tested with:
![image](https://user-images.githubusercontent.com/10697674/143562762-131e7803-0227-4244-8e18-de4d8be11022.png)

Currently I have no idea why the installation on your computer is going sideways. Are there any TaggingKit log files in your %TEMP% directory you can send me?

**Reply by roy6n23** [reply]
Thank you for your reply. May I ask where is the TaggingKit log file? Does it exist in C:\Windows\Temp ? But I cannot find any .tmp file name related with TaggingKit.

Best regards.

**Reply by WetHat** [reply]
The log files are in %TEMP% which is usually something like  `C:\Users\<user>\AppData\Local\Temp`, where `<user>` is your username. The log files are named something like `taggingkit_8D9B273B2D6857C.log`. The hexadecimal part of filename the is used to make it unique. So you would search  for `taggingkit_*.log`. If no such file is in %TEMP%,  the add-in code was never initialized and something else prevented the add-in from being loaded.

**Reply by roy6n23** [reply]
I have solved it!!! In fact, not my credit, because I found the original OneNote login function also appeared abnormal, I understood that may be my software problems, not your add-in problems, so I just office components constantly reinstalled and uninstalled, after three times, finally succeeded. Thank you for your patient answer, hope you all go well!

**Reply by WetHat** [reply]
Congratulations for solving this issue! Repairing an Office installation is not a lot of fun. Enjoy the TaggingKit...

**Comment by WetHat** [comment]
There is a service called 'dllhost' which is needed to load .net add-ins (which the TaggingKit is). Sometimes that service gets stuck. The easiest way to recover  from  that is to **reboot** the computer and them open OneNote again. If the TaggingKit dows not show up after that please try the steps in   [Tagging Kit Not Shown in Ribbon](https://github.com/WetHat/OnenoteTaggingKit/wiki/Tagging-Kit-Not-Shown-in-Ribbon).

Let me know how that goes.
